src/lib/mqtt/MqttConnectionManager.cpp

```diff
@@ -389,6 +417,11 @@ bool MqttConnectionManager::sendMqttNotification(const std::string& host, int po
     return false;
   }
 
+  if (timeoutInMilliseconds > 0)
+  {
+    curl_easy_setopt(curl, CURLOPT_TIMEOUT_MS, timeoutInMilliseconds);
```

Comment:
This belong to the libcurl library, the one used to send HTTP notifications (MQTT notifications are sent using mosquitto library). However, note you are here in sendMqttNotification() function, so this doesn't make sense...

Reply:
Fixed in [a06ac67](https://github.com/telefonicaid/fiware-orion/pull/4313/commits/a06ac672070c368a6dddf831bedabae222076981)